pkg/migrations/sql/idempotent/base.sql

```diff
@@ -1609,7 +1602,9 @@ BEGIN
     IF older_than IS NULL THEN
         -- even though there are no new Ids in need of deletion,
         -- we may still have old ones to delete
-        PERFORM SCHEMA_CATALOG.delete_expired_series(metric_table, ran_at);
+        SELECT current_epoch, last_update_time INTO present_epoch, last_updated FROM
```

Comment:
This has to happen in a different transaction than the call to delete_expired_series. I suggest moving this up to right before the commit on line 1600